rds-replica/main.tf

```diff
@@ -21,7 +21,8 @@ resource "aws_db_instance" "rds" {
   parameter_group_name            = var.custom_parameter_group_name == "" ? data.aws_db_instance.master.db_parameter_groups[0] : var.custom_parameter_group_name
   enabled_cloudwatch_logs_exports = var.enabled_cloudwatch_logs_exports
   backup_retention_period         = var.backup_retention_period
-
+  auto_minor_version_upgrade      = false
```

Comment:
I think this should be configurable.

Reply:
Nope, The auto-minor-version-upgrade value is not valid for the Read Replica

Reply:
That's not true, just tested for both Postgres and mysql, auto minor upgrades can be set and are enabled by default (via the AWS dashboard)

![image](https://user-images.githubusercontent.com/33081/63688030-851add00-c806-11e9-8562-2027484f381b.png)